### PR TITLE
Facebook import

### PR DESCRIPTION
**Pull request by henmyhre**
Added functionality to upload and register facebook donations. See https://github.com/stiftelsen-effekt/effekt-admin/pull/226 for more info.

### CODE REVIEW

File: src/routes/facebook.ts [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+const authMiddleware = require("../custom_modules/authorization/authMiddleware");
+const donationHelpers = require("../custom_modules/donationHelpers");
+
+function throwError(message) {
+  let error = new Error(message);
+  (error as any).status = 400;
+  throw error;
+}
+
+router.get("/payments/all", async (req, res, next) => {
```
Comment (philiphand) [inline src/routes/facebook.ts:17]: Mangler authorization?

**Review by philiphand — CHANGES_REQUESTED** [review]
Ser bra ut! Men ser ut som den ene routen mangler auth

**Review by fellmirr — CHANGES_REQUESTED** [review]
Looking very snazzy, only one small question :)

File: src/routes/facebook.ts [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+        const donorID = ID;
+        const donor = await DAO.donors.getByID(ID);
+
+        if (!donor.ssn) await DAO.donors.updateSsn(donorID, ssn);
+        await DAO.donors.updateNewsletter(donorID, newsletter);
+
+        await DAO.facebook.registerPaymentFB(donorID, paymentID);
+      }
+
+      await mail.sendFacebookTaxConfirmation(email, full_name, paymentID);
```
[the remark this answers is not in the document]
Comment (fellmirr) [reply src/routes/facebook.ts:72]: Misunderstanding, used by the tax form

**Pull request merged**